apps/api-example/src/nestapp/gateway.module.ts

```diff
@@ -0,0 +1,46 @@
+import {IntrospectAndCompose, RemoteGraphQLDataSource} from '@apollo/gateway'
+import {ApolloGatewayDriver, ApolloGatewayDriverConfig} from '@nestjs/apollo'
+import {Module} from '@nestjs/common'
+import {GraphQLModule} from '@nestjs/graphql'
+
+@Module({
+  imports: [
+    GraphQLModule.forRoot<ApolloGatewayDriverConfig>({
+      driver: ApolloGatewayDriver,
+      server: {
+        introspection: true,
+        sortSchema: true,
+        path: 'v1',
+        cache: 'bounded',
+        playground: process.env.NODE_ENV === 'development',
```

Comment:
Here also use the filed from the config "apolloPlayground"

Reply:
Oh I've tried, but I cannot use `GraphQLModule.forRootAsync` seems it's not accepting driver property.... strange. Probably bug with graphql/nestjs.

Reply:
Nvm, actually I can. Just strange driver was put outside of the useFactory returned options.